backend/config/corpora-api.yml

```diff
@@ -780,6 +791,18 @@ components:
           enum: [REMIX, ORIGINAL]
         s3_uri:
           type: string
+    problem:
+      type: object
+      description: >-
+        The error response shape based on
```

Comment:
Will this actually show up as markdown in the spec? Could just simplify to "Error message container for HTTP APIs.

Reply:
yes it shows up as markdown, but i can simplify to that.

Reply:
Ah good to know! Thanks!